constants_local.xml

```diff
@@ -85,7 +85,7 @@
         <LEGAL_GAS_PRICE_IP>127.0.0.1</LEGAL_GAS_PRICE_IP>
     </gas>
     <gossip>
-        <BROADCAST_GOSSIP_MODE>true</BROADCAST_GOSSIP_MODE>
+        <BROADCAST_GOSSIP_MODE>false</BROADCAST_GOSSIP_MODE>
```

Comment:
One question here: does this mean that for local testing we will disable gossip from now on?

Reply:
@Gnnng also if we don't use `-t large` when bootstrapping, does the cloud test take the settings from this xml?

Reply:
Should be fine.  I think @iantanwx has frequently needed to use this setting for his small-scale cloud tests

Reply:
Yes I usually switch it to the naive multicast for the small e2e tests.